packages/api/test/feeds.spec.ts

```diff
@@ -29,12 +29,15 @@ describe.skip('feeds', function () {
 
     const server = await createServer(
       {
-        feedRepository: new FeedRepository(dataFeeds),
-        resultRequestRepository: new ResultRequestRepository(db, dataFeeds)
+        feedRepository: new FeedRepository(await dataFeeds),
+        resultRequestRepository: new ResultRequestRepository(
+          db,
+          await dataFeeds
+        )
       },
       {
-        dataFeedsConfig: dataFeeds,
-        networksConfig: networksconfig
+        dataFeedsConfig: await dataFeeds,
```

Comment:
Spite of we are fetch the config from Github, I'd mock the call to have deterministic tests